addons/sale_crm/crm_lead.py

```diff
@@ -1,5 +1,7 @@
 from openerp.osv import osv, fields
+from openerp.addons.crm.crm_lead import CRM_LEAD_FIELDS_TO_MERGE
 
+CRM_LEAD_FIELDS_TO_MERGE.append('quotation_ids')
```

Comment:
While you are at it, could you move this global variable to the crm_lead class as a class variable in crm/crm_lead.py, and update its udpate here ? This variable looks more like a class variable than a global variable to me.